.github/workflows/containerd.yml

```diff
@@ -0,0 +1,91 @@
+name: critest
+on: [push, pull_request]
+
+jobs:
+  #
+  # Run CRI tests against containerd
+  #
+  linux-build-and-test-containerd:
+    strategy:
+      matrix:
+        version: [master, v1.4.0-beta.1]
+    name: containerd ${{matrix.version}} (linux amd64)
```

Comment:
What do you like more:
```
critest / containerd master (linux amd64) (pull request)
```
or
```
critest / containerd master / linux amd64 (pull request)
```

Reply:
Good pint @saschagrunert. I think the second version is more symmetrical. Updated to mirror that.